ref/api-reference.md

```diff
@@ -1,7 +1,174 @@
-The documentation in this section provides a reference for the following APIs provided by Anbox Cloud:
+All communication between AMS and its clients happen using a RESTful API over HTTP. This API is encapsulated over either TLS (for remote operations) or a Unix socket (for local operations).
+
+## APIs and their structure
+Anbox Cloud provides the following APIs:
+
 * AMS HTTP API
 * Anbox Cloud HTTP API
```

Comment:
What's the `Anbox Cloud HTTP API`? do you mean the Anbox HTTP API that was referred below?

Reply:
Yes, by terminology convention, it is better to avoid using just "Anbox" anywhere and use the full product name "Anbox Cloud". I have corrected both instances to read "Anbox Cloud HTTP API".